src/main/java/org/jboss/windup/operator/controller/WindupController.java

```diff
@@ -0,0 +1,80 @@
+package org.jboss.windup.operator.controller;
```

Comment:
_Comment to further review the code, NOT mandatory for this PR_
As state in other comments, this package seems more related to `Watcher`s

Reply:
in the operators naming convention they are controllers, now it's a watcher but in the future it will be a SharedInformer that uses queues, now it's a watcher because informers have an issue in the current used version